src/api/apps.ts

```diff
@@ -11,6 +15,18 @@ export const getApps = async ({ API_HOST, axios }: PartialQueryConfigForApi) =>
       .then(({ data }) => data),
   );
 
+export const getMostUsedApps = async ({
+  API_HOST,
+  axios,
+}: PartialQueryConfigForApi) =>
+  verifyAuthentication(() =>
+    axios
+      .get<
+        { url: string; name: string; nbr: number }[]
```

Comment:
I would use `count` rather than `nbr`. I get this should apply to the backend as well.